### PR TITLE
un-sudo'ing mrbayes

### PR DESCRIPTION
**Pull request by markiskander**
Removed sudo, added prefix to configure

### CODE REVIEW

**Comment by ericenns** [comment]
Instead of copying 'src' to installdir, you can use the 'change_directory' action. Then please use the 'make_install' action.


e.g. https://github.com/galaxyproject/tools-iuc/blob/86a44bf15f8184b690ee1af98aa1d5f86ad2fb3d/packages/package_ccat_3_0/tool_dependencies.xml). Then you can use the

**Comment by markiskander** [comment]
Properly installs and tested with planemo serve

File: packages/package_mrbayes_3_2_6/tool_dependencies.xml [hunk 1]
```diff
@@ -4,29 +4,14 @@
...
-		  <destination_directory>$INSTALL_DIR/examples</destination_directory>
-		</action>
-                <action type="move_directory_files">
-      <source_directory>src</source_directory>
-      <destination_directory>$INSTALL_DIR/src</destination_directory>
-    </action>
+                <action type="change_directory">src</action>
                 <action type="set_environment">
-                <environment_variable name="PATH" action="prepend_to">$INSTALL_DIR/src</environment_variable>
+                  <environment_variable name="PATH" action="prepend_to">$INSTALL_DIR/src</environment_variable>
```
Comment (ericenns) [inline packages/package_mrbayes_3_2_6/tool_dependencies.xml]: This isn't needed

File: packages/package_mrbayes_3_2_6/tool_dependencies.xml [hunk 2]
```diff
@@ -4,29 +4,14 @@
...
                 </action>
 
-                <action type="shell_command">(cd $INSTALL_DIR/src; autoconf)</action>
-                <action type="shell_command">(cd $INSTALL_DIR/src; ./configure --with-beagle=no)</action>
-                <action type="shell_command">make -C $INSTALL_DIR/src</action>
-                <action type="shell_command">sudo make install -C $INSTALL_DIR/src</action>
+                <action type="shell_command">autoconf</action>
```
Comment (ericenns) [inline packages/package_mrbayes_3_2_6/tool_dependencies.xml]: please change this to the action autoconf.

e.g. https://github.com/galaxyproject/tools-iuc/blob/508edb578630628be38be6dc9c97e9a3cd0992d2/packages/package_gnu_awk_4_1_0/tool_dependencies.xml

**Pull request merged**